errors/errors.go

```diff
@@ -119,6 +124,17 @@ func NewErrorWithExtras(_msg string, _cause error, _fatal bool, _extras map[stri
 	return _err.WithStack(err)
 }
 
+// NewErrorToIgnore returns an error that informs loggers to ignore it
+func NewErrorToIgnore(_msg string, _cause error, _fatal, ignore bool) error {
```

Comment:
Ignorable error cannot be fatal as well. So let''s set fata to false by default here.

Reply:
> Ignorable error cannot be fatal as well. So let''s set fata to false by default here.

Pushed the change in a new commit.